infrateam_first_responder.md

```diff
@@ -66,7 +72,7 @@ If on a Mac, you will get better results if you stay in the same "space" as the
 
 Use the `sdr-deploy` script to deploy all infrastructure projects (with **required additional deploys** noted below) via capistrano to deployed environments: https://github.com/sul-dlss/sdr-deploy
 
-Note that you will need to be sure you can ssh into each of the VMs from wherever you are running the deploy script.
+Note that you will need to be sure you can ssh into each of the VMs from wherever you are running the deploy script.  You can do this using the `--checkssh` flag.
```

Comment:
This syntax has now changed. We can either update the syntax here or maybe it's sufficient to link to sdr-deploy's own documentation in case it changes again: https://github.com/sul-dlss/sdr-deploy/#check-your-ssh-connection-to-all-servers